src/APICalls/APICalls.js

```diff
@@ -0,0 +1,53 @@
+const theEndpoint = `https://bangwords-api.herokuapp.com/`;
+
+const theCalls = {
+    joinGame(role) {
+        let theBody = {
+            act: "join",
+            method: 'POST',
+            headers: {
+                "Content-Type": "application/json"
+            },
+            body: JSON.stringify(role)
+        }
+        return fetch(`${theEndpoint}`, theBody)
+            .then(res => res.json())
+            .then(res => console.log(res))
+            .catch(err => console.log(err))
```

Comment:
``` JavaScript
.then(res => res.json())
.then(res => console.log(res))
.catch(err => console.log(err))
```

I suppose it's harmless, but can we consider using only one `.then` here? Can we also add a little error processing?

``` JavaScript
.then( res => {
  if (res.ok) {
    return res.json();
  } else {
     return 'error'
  }
})
```

If we want to console log the `json` data, we can do that wherever we call it, no? I find it simpler to keep the console logs in the files we're working with directly (makes it easier to keep good code hygiene!). 

The error catching makes it ridiculously simple to test if we got an error back from the server.